build.sh

```diff
@@ -5,7 +5,7 @@ source ./dev_up.sh
 cd reactivexcomponent
 
 echo Running lint...
```

Comment:
Optional: I think it could be interesting to get separate steps in travis (build, lint, test, and package). It's easier to investigate in travis when something goes wrong
Just an idea for a next PR

Reply:
https://github.com/xcomponent/ReactiveXComponent.py/issues/34